integration-tests/micrometer-native/src/main/java/io/quarkus/it/micrometer/native_mode/MessageResource.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.it.micrometer.native_mode;
+
+import java.lang.management.ManagementFactory;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import javax.management.*;
```

Comment:
wildcard imports are not used throughtout the codebase.

Reply:
It was also extraneous. ;) I removed it.